addon/-private/engine-ext.js

```diff
@@ -52,7 +51,6 @@ Engine.reopen({
 
       registry.register('template:-outlet', topLevelViewTemplate);
       registry.injection('route', '_topLevelViewTemplate', 'template:-outlet');
-      registry.register('view:-outlet', OutletView);
```

Comment:
Looking through the Ember 2.6 code, seems like it is redundant, as it gets registered via the super of `buildRegistry`.